modules/builds-source-secrets-entitlements.adoc

```diff
@@ -12,6 +12,16 @@ Builds that use Red Hat subscriptions to install content must include the entitl
 
 You must have access to Red Hat entitlements through your subscription, and the entitlements must have separate public and private key files.
 
+
+[TIP]
+====
+When you perform an Entitlement Build using {op-system-base-full} 7 you must have the following command in your Dockerfile before run any `yum` commands:
+[source,terminal]
+----
+$  oc RUN rm /etc/rhsm-host
```

Comment:
Is the `$  oc` part supposed to be here? I thought this was intended to be a Dockerfile instruction.